webapp/components/_new/generic/UserAvatar/UserAvatar.vue

```diff
@@ -2,18 +2,18 @@
   <div :class="['user-avatar', size && `--${this.size}`]">
     <span class="initials">{{ userInitials }}</span>
     <base-icon v-if="isAnonymous" name="eye-slash" />
-    <img
-      v-else
-      :src="user.avatar | proxyApiUrl"
-      class="image"
-      @error="$event.target.style.display = 'none'"
-    />
+    <responsive-image v-else :image="user.avatar" class="image" sizes="120px" />
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-03-30T21:35:47Z" title="Monday, March 30th 2020, 11:35:47 pm +02:00">Mar 30, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -2,18 +2,18 @@
   <div :class="['user-avatar', size && `--${this.size}`]">
     <span class="initials">{{ userInitials }}</span>
     <base-icon v-if="isAnonymous" name="eye-slash" />
-    <img
-      v-else
-      :src="user.avatar | proxyApiUrl"
-      class="image"
-      @error="$event.target.style.display = 'none'"
-    />
+    <responsive-image v-else :image="user.avatar" class="image" sizes="120px" />
```


So this is the only place where I decided to set a `sizes`. It's fixed to one size only. I went for 120px which is a little more than the 114px avatar image on the profile page. The avatar image on the post card is 34px wide on my machine. Good @alina-beck ?

If we want to keep that, maybe we want to query for 120px only in the respective graphql request.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-01T07:04:47Z" title="Wednesday, April 1st 2020, 9:04:47 am +02:00">Apr 1, 2020</time>_

---


and giving it `34px, 120px` sizes would do anything because the size it uses is determined by the viewport width, correct? I guess it's not any different on mobile either, as far as I understand how it works.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-04-09T10:57:25Z" title="Thursday, April 9th 2020, 12:57:25 pm +02:00">Apr 9, 2020</time>_

---


Yes, you the developer must give the browser a hint how wide an image is going to be for a given viewport.